examples/wflow_update_water_demand.yml

```diff
@@ -26,9 +26,9 @@ setup_lulcmaps_with_paddy:
     - null
     - null
   paddy_waterlevels:
-    "h_min": 20
-    "h_opt": 50
-    "h_max": 80
+    "demand_paddy_h_min": 20
+    "demand_paddy_h_opt": 50
+    "demand_paddy_h_max": 80
```

Comment:
It's up for debate but I think function arguments can stay the same (the function is called irrigation and dict name is paddy_waterlevels)